code/modules/clothing/masks/gasmask.dm

```diff
@@ -165,12 +178,12 @@
 	clothing_flags = MASKINTERNALS
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
-
+	species_restricted = FLAG_GROD
 /obj/item/clothing/mask/gas/carp
```

Comment:
improper spacing, will break shit. there needs to be at least one line between the definitions,
there's a few in this file

Reply:
This is straight up wrong. It won't break anything. The DM compiler doesn't care about extra lines.

Extra lines are just for readability.